### ISSUE TITLE
Implement OIDC IDP Logout

### ISSUE THREAD

**Issue by jjoyce0510** [linkedin/datahub#2735]
**Is your feature request related to a problem? Please describe.**
When leveraging OIDC for authentication, clicking logout only logs out the DataHub session and not the IdP session. We need to implement support for calling the IdP to logout. 

**Describe the solution you'd like**
Call the IdP on logout to kill the session. 

**Describe alternatives you've considered**
N/A

**Additional context**
As reported by a few folks in the community.

**Comment by jjoyce0510** [comment]
This should be in progress - A community member has already implemented in, just waiting on the official PR

**Comment by maggiehays** [comment]
We are now managing feature requests through our new portal to improve visibility & prioritize work into our public roadmap. If this request is still relevant, please create a new issue so we can gauge current interest from the Community & prioritize work accordingly.

We'll close out this issue in the meantime. Feel free to reach out to me on DataHub Slack with any questions!
